kik_unofficial/client.py

```diff
@@ -84,7 +84,8 @@ def __init__(
         self.android_id = android_id
 
         self.callback = callback
-        self.callback._on_client_init(self)
+        if self.callback:
```

Comment:
If someone creates a bot without a callback I'm in a doubt it could do something useful (?)

Reply:
I agree, when not using callbacks youre not going to profit from this. But for those that use callbacks can use them without having to find a workaround to add the Client to the Callbacks context.

```py
callback = InteractiveChatClient()
client = KikClient(callback=callback,...)
```
in this [example](https://github.com/tomer8007/kik-bot-api-unofficial/blob/new/examples/interactive_client.py) you cant add the client to the callback, after the client is created, because the KikClient accesses callback before it returns